App/views.py

```diff
@@ -0,0 +1,24 @@
+from django.http import HttpResponse
+from django.shortcuts import render
+import json
+import requests
+
+def index(request):
+    return HttpResponse("Hello, world. You're at the polls index.")    
+
+def main(request):
+	if(request.method=='POST'):
+		usr = request.POST['username']  #Here, we get the useranme fetched from html page
+		URL = "https://en.wikipedia.org/w/api.php?action=query&format=json&list=usercontribs&ucuser="
+		URL = URL + usr
+		r = requests.get(URL)  #Recieved data in json-format
+		lst = json.loads(r.text)
+		sub_lst = lst['query']['usercontribs'] #Contribution data extracted
```

Comment:
The API might respond with an error, in which case this key might not exist. (You can test with a username containing `<`.)

Reply:
Resolved.